jina/executors/indexers/query/__init__.py

```diff
@@ -57,3 +57,13 @@ def delete(self, keys: Iterable[str], *args, **kwargs) -> None:
         .. # noqa: DAR101
         """
         self._log_warn()
+
+    def is_exist(self) -> bool:
+        """
```

Comment:
Since all the boilerplate and the extra code in the Executor will be removed, explicitly overwrite how the write handler and query handler are computed. Or do some alternative naming.

Just overdiding is_exist to False is too hacky. It invalidates the semantics

Reply:
This is not used. I simply get the write handler